hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/StoreFileInfo.java

```diff
@@ -107,7 +108,7 @@
 
   private RegionCoprocessorHost coprocessorHost;
 
-  // timestamp on when the file was created, is 0 and ignored for reference or link files
+  // change to use the createdTimestamp of reference or link files , will it create some problem later ?
```

Comment:
Is this comment correct? The createTimeStamp is gotten once only on construction. YOu are not changing the behavior that I can see. Should createTimestamp be made final as in private final long createdTimestamp?

Reply:
It is ok to make createdTimestamp to be final  i think.